### PR TITLE
feat: added the usage of utils for status messages and codes

### LINKED ISSUE #27: 🐞Bug: Using the `STATUS` utils instead of direct numbers
### Description

Currently, we have [Status codes](https://github.com/MilanCommunity/Milan-Backend/blob/c2ee6b94d30953e435a7b6d1fb550dab349f180f/utils/Status.js#L1) and [Status messages](https://github.com/MilanCommunity/Milan-Backend/blob/c2ee6b94d30953e435a7b6d1fb550dab349f180f/utils/Status.js#L28) that are configured for each of those codes that are being used. The goal is to replace any codes we find, with these , like `200` might be replaced with `STATUSCODE.OK` and then the message for some cases saying "Internal server error" might be replaced with `STATUSMESSAGE.INTERNAL_SERVER_ERROR` and so on.

Please ensure that this is done for all the codes in the project.

### Screenshots

image

### Browser 🥦

Google Chrome

### Checklist ✅

- [X] I checked and didn't find similar [issue](https://github.com/MilanCommunity/Milan-Backend/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/MilanCommunity/Milan-Backend/blob/main/CONTRIBUTING.md)
- [ ] I am willing to work on this issue (blank for no).

### PR DESCRIPTION
**Pull request by cu8code**
chore(refactor): refactored legacy codes for server.

closes #27 

### 👷🏻 Changes made  
refactor code to use Status code & Status message.

### CODE REVIEW

**Comment by cu8code** [comment]
@tamalCodes please review this

File: routes/user/Auth.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ router.post("/update", async (req, res) => {
     const { email, oldPassword, newPassword } = req.body;
     const existingUser = await User.findOne({ email: email });
     if (!existingUser) {
-      return res.status(404).json({ message: "User not found" });
+      return res.status(STATUSCODE.NOT_FOUND).json({ message: "User not found" });
```
Comment (tamalCodes) [inline routes/user/Auth.js:95]:
```suggestion
      return res.status(STATUSCODE.NOT_FOUND).json({ message: STATUSMESSAGE.USER_NOT_FOUND });
```

**Review by tamalCodes — CHANGES_REQUESTED** [review]
- Alot of places are still using messages like `message: "Internal Server Error"` instead of that they can use `STATUSMESSAGE.INTERNAL_SERVER_ERROR` 
 - Make sure all the places are using `STATUSCODE` and `STATUSMESSAGE` except for those which has custom messages.

With those changes done, we would be good to go.
Sorry about the delay.

**Comment by tamalCodes** [comment]
# Notice 🚩

Hi there, unfortunately, I will have to close this Issue/PR, as this issue/PR has been inactive for quite a while and there's no work going on for the same.

There might be more Issues/PR that are waiting for this to be completed - so it is better to pace things up. I generally spend/wait for 3 weeks before marking any issue as stale and closing it.

If you think that this is a mistake, please drop an email at milanhub.org@gmail.com, and we will get back to you.

Regards, 
Team Milan.

**Pull request closed without merging**